raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -725,16 +759,16 @@ contract TokenNetwork is Utils {
 
         require(verifyBalanceHashData(
```

Comment:
`verifyBalanceHashData` is also never used outside the context of settlement, right? So it could use `(Participant)SettleInput` as the sole argument.